extensions/amp-consent/0.1/consent-ui.js

```diff
@@ -151,6 +167,90 @@ export class ConsentUI {
     const iframe = this.parent_.ownerDocument.createElement('iframe');
     iframe.src = assertHttpsUrl(promptUISrc, this.parent_);
     iframe.setAttribute('sandbox', 'allow-scripts');
+    const {classList} = iframe;
+    classList.add('i-amphtml-consent-fill');
+    // Append iframe lazily to save resources.
     return iframe;
   }
+
+  /**
+   * Create the default placeholder
+   * @return {!Element}
+   */
+  createPlaceholder_() {
+    // TODO: Allow publishers to provide placeholder upon request
```

Comment:
nit: Can we add your github username to the TODOs? I find it helpful when looking through the code when wanting to ask a question later on. Unless these will be solved in the next PR 😄

Reply:
Good point! Added.